### PR TITLE
android: Add licenses page

### PR DESCRIPTION
**Pull request by t895**
Please let me know if I should add any more licenses now or in the future.

### CODE REVIEW

File: src/android/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -346,4 +347,511 @@
     <string name="use_black_backgrounds">Use Black Backgrounds</string>
     <string name="use_black_backgrounds_description">When using the dark theme, apply black backgrounds.</string>
 
+    <!-- Licenses screen strings -->
```
Comment (FearlessTobi) [inline src/android/app/src/main/res/values/strings.xml:350]: can we make it so these are not translatable?

Comment (t895) [reply src/android/app/src/main/res/values/strings.xml:350]: For all of them or just the license text?

Comment (FearlessTobi) [reply src/android/app/src/main/res/values/strings.xml:350]: I think it's fine if everything in the license section except the Strings `licenses`  and `licenses_description` stays in English.
I have never seen it localized and it just created additional work on Transifex.

Comment (AbandonedCart) [reply src/android/app/src/main/res/values/strings.xml:350]: Licenses aren't translated.

**Review by bunnei — APPROVED** [review]

**Pull request merged**